language-server/modules/langserver-core/src/test/resources/codeaction/implement-all/config/implementAllObj1.json

```diff
@@ -16,7 +16,7 @@
                             "character": 0
                         }
                     },
-                    "newText": "\n    public function pushText2(string text) returns error? {\n        return error (\"\");\n    }\n"
+                    "newText": "\n    public function pushText2(string text) returns error? {\n        return ();\n    }\n"
```

Comment:
For the return use-case, it is not mandatory to return `()`. Instead we can just use return; Shall we fix this?

Reply:
@nadeeshaan For optional types, the idea was to use `()` as the default value. Do you mean to change it to `""`?
This will also change the default value of optional record fields to be empty.
Eg:
```
type LinkedList record {|
    string value;
    LinkedList? next;
|};

LinkedList l0 = {value: "", next:};
```
Or to fix it in `generateFunction()` to have `return;` whenever a function returns a nil?

Reply:
The best option here is to add the fix to the `generateFunction()` method